CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## master
 
 * Add your own contributions to the next release on the line below this with your name.
+- [ASStackLayoutSpec] Fix flex wrap overflow and add lineSpacing property. [Flo Vouin](https://github.com/flovouin)
```

Comment:
Since line spacing is a significant addition, I'd suggest to mention it on a separate line so people won't overlook it. It's your call though.